plan_export.go

```diff
@@ -58,11 +57,11 @@ const (
 
 // PlanExportStatusTimestamps holds the timestamps for plan export statuses.
 type PlanExportStatusTimestamps struct {
-	CanceledAt time.Time `json:"canceled-at"`
-	ErroredAt  time.Time `json:"errored-at"`
-	ExpiredAt  time.Time `json:"expired-at"`
-	FinishedAt time.Time `json:"finished-at"`
-	QueuedAt   time.Time `json:"queued-at"`
+	CanceledAt string `jsonapi:"attr,canceled-at"`
+	ErroredAt  string `jsonapi:"attr,errored-at"`
+	ExpiredAt  string `jsonapi:"attr,expired-at"`
+	FinishedAt string `jsonapi:"attr,finished-at"`
+	QueuedAt   string `jsonapi:"attr,queued-at"`
```

Comment:
This is fixed [here](https://github.com/hashicorp/jsonapi/pull/1), will update this comment soon.